modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/WindowStage.java

```diff
@@ -613,8 +594,7 @@ private void applyFullScreen() {
                 // indicating that the fullscreen request came from an input
                 // event handler.
                 // If not notify the stageListener to reset fullscreen to false.
-                final boolean isTrusted = isTrustedFullScreen();
-                if (!isTrusted && !fullScreenFromUserEvent) {
+                if (!fullScreenFromUserEvent) {
```

Comment:
is this correct?

do I understand that the effective value for isTrusted is true, which makes the condition that the if tests false, so the (new) lines 598-599 would never execute.

Reply:
Good catch. You are right. The entire block should be removed. As it is, this will incorrectly limit the set of keys that can be accessed in full-screen mode.

This might be why Lukasz thought that `fullScreenFromUserEvent` was still needed.

Reply:
Indeed good catch. Monday brain now wonders what Friday brain was thinking 😉